src/test/java/it/EndpointIT.java

```diff
@@ -23,7 +23,7 @@
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.Response;
 
-public class EndpointTest {
+public class EndpointIT {
```

Comment:
The special naming is needed for the files that have test annotations that you want failsafe or surefire to find.